internal/util/httpserver.go

```diff
@@ -24,3 +26,22 @@ func StartMetricsServer(c *Config) {
 		FatalLogMsg("failed to listen on address %v: %s", addr, err)
 	}
 }
+func addPath(name string, handler http.Handler) {
+	http.Handle(name, handler)
+	DebugLogMsg("DEBUG: registered profiling handler on /debug/pprof/%s\n", name)
```

Comment:
nit: drop `DEBUG:` from this message, it will be tagged with debug level already too through `DebugLogMsg`

Reply:
okay sure.